src/agentscope/configs/model_config.py

```diff
@@ -113,3 +117,6 @@ class PostApiCfg(CfgBase):
     """The key of the prompt messages in `requests.post()`,
     e.g. `request.post(json={${messages_key}: messages, **json_args})`. For
     huggingface and modelscope inference API, the key is `inputs`"""
+
+    budget: float = _DEFAULT_API_BUDGET
```

Comment:
It's confused to set budget for POST API. Since we don't know what's the model behind POST service, we also don't know how to compute its budget.

Reply:
It is better to keep it, maybe it will be used in the future